src/applications/disability-benefits/686c-674/config/helpers.js

```diff
@@ -52,10 +52,8 @@ export const ServerErrorAlert = (
     </p>
     <p>
       If you get this error again, please call the VA.gov help desk at{' '}
-      <a href="tel:8446982311" aria-label="8 4 4. 6 9 8. 2 3 1 1.">
-        844-698-2311
-      </a>{' '}
-      (TTY: <Telephone contact={CONTACTS['711']} pattern={PATTERNS['911']} />
+      <Telephone contact="8006982411">800-698-2411</Telephone> (TTY:{' '}
```

Comment:
Would it make sense to add this phone number to CONTACTS in the Telephone component?

Reply:
Definitely worthwhile. I'll create a ticket for doing so 👍

Reply:
It appears Kelson already just pushed up [a PR](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/pull/490/files) that does just this 😄